runtime/runtime-params-estimator/src/action_costs.rs

```diff
@@ -317,6 +343,7 @@ pub(crate) fn deploy_contract_byte_send_sir(ctx: &mut EstimatorContext) -> GasCo
     ActionEstimation::new_sir(ctx)
         .add_action(deploy_action(ActionSize::Max))
         .inner_iters(1) // circumvent TX size limit
+        .min_gas(1_000_000) // 1 ns accuracy for per-byte cost
```

Comment:
Rather than a comment saying what this is, a more useful one would be an explanation of _why_. In particular it would be useful to see a typical cost we see without the `min_gas` threshold for each action with `min_gas`.

Reply:
Yeah, these are very nice now!